README.md

```diff
@@ -18,6 +18,29 @@ If you are using TypeScript you can install the accompanying types
 npm install --save-dev @types/slug
 ```
 
+## Differences between `slug` and `slugify` packages
+
+I also contribute to a similar package called [`slugify`](https://github.com/simov/slugify).
+Here are the key differences.
+
+- **Defaults:** `slug` has the `lower` option enabled by default, lowercasing all slugs
+  (`'On SALE'` becomes `'on-sale'`).  
+  `slugify` has the `lower` option disabled by default (`'On SALE'` becomes `'On-SALE'`).
+- **IE11 Support:** `slug` currently still supports IE11
+  (_However, [support will be dropped June 2022](https://github.com/Trott/slug/blob/beta/CHANGELOG.md)!_).  
+  `slugify` dropped support for IE11 some time ago.
+- **Currency:** `slug` removes currency symbols (`'$100'` becomes `'100'`).  
+  `slugify` maps them to words (`$100` becomes `dollar100`).
```

Comment:
It's not just currency, although that's a big one. Slugify has other symbols that it converts to words by default. For example, it converts `<` to `less`. I'm not sure how to explain that succinctly. Maybe just say "currency and other symbols" instead of "currency"?

Reply:
Yeah, "symbols" is a better word for it. We can give multiple examples including the less than and currency.

Reply:
Updated.